conf.yaml

```diff
@@ -1058,7 +1058,7 @@ contents:
               - title:      Enterprise Search Node.js client
                 prefix:     enterprise-search-node
                 current:    *stackcurrent
-                branches:   [ {main: master}, 8.6]
+                branches:   [ {main: master}, 8.7]
```

Comment:
>                branches:   [ {main: master}, 8.7]

Is this correct (i.e. removal of 8.6 instead of just appending 8.6)?

Reply:
Nope! Fixed.